src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+
+  if (days < 3) {
+    sum = days * 40;
```

Comment:
you can also return the final cost in the if-statement, there are no further calculations in this function